Tests/Recurly/Pager_Test.php

```diff
@@ -99,4 +95,10 @@ public function testFromNested() {
     $this->assertEquals(4, count($pager), 'Returns correct count');
     $this->assertIteratesCorrectly($pager, 4);
   }
+
+  public function testCount() {
+    $this->client->addResponse('HEAD', '/mocks', 'pager/head-200.xml');
+    $pager = new Mock_Pager('/mocks', $this->client);
+    $this->assertEquals($pager->count(), 42);
+  }
```

Comment:
I don't think we need this one, it should be covered by the other three scenarios.